BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsEventParams.kt

```diff
@@ -31,5 +31,8 @@ data class AnalyticsEventParams @JvmOverloads constructor(
     val experiment: String? = null,
     val paymentMethodsDisplayed: List<String> = emptyList(),
     val appSwitchUrl: String? = null,
-    val shopperSessionId: String? = null
+    val shopperSessionId: String? = null,
+    val buttonType: String? = null,
+    val buttonOrder: String? = null,
+    val pageType: String? = null
```

Comment:
We don't technically need doc strings for this class, but since we have them for all other params lets add them for consistency and for our future reference